src/rviz/yaml_config_writer.cpp

```diff
@@ -152,7 +152,9 @@ void YamlConfigWriter::writeConfigNode(const Config& config, YAML::Emitter& emit
     break;
   }
   default:
+#ifndef _WIN32
     emitter << YAML::Null;
+#endif
```

Comment:
I assume @seanyen originally had written this. Any idea what it's good for @wolfv @traversaro @ooeygui?

Reply:
Without a comment, I'm not sure why it was disabled. Following up internally.

Reply:
@Tobias-Fischer @ooeygui I didn't recall I make this type of change. Can you remind me where did you see this change?

Reply:
Did it originate here? https://github.com/RoboStack/ros-noetic/commit/8c9625a0ca195ed8235c4e4ec96329d0e49a4eb7#diff-1258ec9b02f6cb78ef3f05f14b086a35379b4137213fe83cabe964bb1bcb0b64

Reply:
Yeah this is where is originates. Maybe it was @wolfv?

Reply:
It might have been me who done it :)
The problem is that there will be a missing symbol on Windows. Somehow the libyaml doesn't properly export the `YAML::Null` symbol on Windows IIRC.

Reply:
somehow this symbol: https://github.com/jbeder/yaml-cpp/blob/4564d9a1314e8dbd01ba769b94133b9eba4f36ad/include/yaml-cpp/null.h#L16 

seems to be missing from Windows DLLs. One could check the package here if it's still the case https://anaconda.org/conda-forge/yaml-cpp/files

Reply:
Just a guess, perhaps `YAML::Null` should be marked as `inline`?

Reply:
This should be fixed upstream then...

Reply:
Upstream issue: https://github.com/jbeder/yaml-cpp/issues/950

Reply:
I think this is the last remaining comment. As this is an upstream bug, would you be happy to just add a comment like I did in https://github.com/ros-visualization/rviz/pull/1636/commits/fae0c72508645f16d89240666e47d957fb7d71a9? We can remove the workaround once the issue is resolved upstream.

Reply:
:+1: